src/components/subscription-choices/subscription-choices.scss

```diff
@@ -0,0 +1,90 @@
+@use '../../compiled/tokens/scss/breakpoint';
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/size';
+@use '../../mixins/a11y';
+@use '../../mixins/theme';
+
+///
+/// Theme properties
+///
+
+// Light theme
+@include theme.props {
+  --theme-color-background-subscription-form: #{color.$base-white};
+}
+
+// Dark theme
+@include theme.props(dark) {
+  --theme-color-background-subscription-form: #{color.$brand-primary};
+}
+
+// Alternate theme
+@include theme.props(alternate) {
+  --theme-color-background-subscription-form: #{color.$base-gray-lighter};
+}
+
+///
+/// Subscription Choices
+///
+
+.c-subscription-choices {
+  position: relative;
+}
+
+.c-subscription-choices__controls-wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+  margin: size.$rhythm-default 0;
+
+  .c-button-swap {
+    flex-grow: 1;
+  }
+
+  .c-button-swap__subscribe {
+    display: flex;
+  }
```

Comment:
It's not a good idea to target one component class name from another like this. You might consider...

1. Merging `c-button-swap` into this component (since this is the only context for it currently).
2. Adding a unique element class for these things with this component's prefixes so the separation of concerns is evident.
3. Create a new modifier for the `c-button-swap` component so it will work in this type of context.
4. Refactor the button swap component so that these overrides are not necessary.

Reply:
+1

This is a common BEM pattern:

```
<div class="button button--modifier">
  <div class="button__avatar avatar avatar--modifier">
      <image class="button__avatar-image avatar__image">
```

This allows you to semantically separate the styles that originate from alpha and beta, while still applying styles from both to the same element.

The only time this becomes a concern is if they're in conflict, then you need to understand the source order of your rendered CSS — but in practice, I've run into this concern very rarely. Typically, as in the example above, you'll be targeting different aspects. Likely, in this example, `button__avatar-image` is dealing with layout concerns like sizing, margins, etc, while `avatar__image` is dealing with things like setting aspect ratio, etc.

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1679/commits/5766e69a1b9456c6eee379e8bad03603e49300de

Reply:
I decided to change this to use CSS grid, which avoids this issue completely. https://github.com/cloudfour/cloudfour.com-patterns/pull/1679/commits/afc52b34481e6ad7e0a367573588b0190a93bfac